resources/[system]/scoreboard/client.lua

```diff
@@ -0,0 +1,118 @@
+RegisterKeyMapping("+scoreboard", "Open the scoreboard.", "keyboard", "z")
+
+RegisterCommand("+scoreboard", function(source, args, rawcommand)
+  if not isShowing then
+    TriggerServerEvent("scoreboard:getPlayers")
+    SendNUIMessage({
+      app = 'CfxScoreboard',
+      method = 'setVisibility',
+      data = true
+    })
+    SetNuiFocus(true, true)
+    SetNuiFocusKeepInput(true)
+    SetCursorLocation(0.5, 0.5)
+    isShowing = true
+    Citizen.CreateThread(function()
+      while isShowing do
+        Citizen.Wait(0)
+        DisableControlAction(0, 1, true)
```

Comment:
What're these control IDs? Magic numbers, too. 😕 

Also, why even disable controls?

Reply:
It disables camera movement and weapon switching via scroll wheel while the scoreboard is open, to allow for scrolling when there are many players, I will add comments to explain these.